### DISCUSSION TITLE
Use Tailscale DNS to setup HTTPS certificates for Home Assistant

### DISCUSSION THREAD

**Discussion by mferstl01** [hassio-addons/app-tailscale#244] [Ideas]
Would it be possible to implement the feature to generate HTTPS ceritificates for Home Assistant via "tailscale cert"?

**Comment by lmagyar** [comment]
It's already implemented, though not released yet, see: #137

Until it is released, you can use my fork, where not only this HTTPS is available, but other useful stuffs also, see: https://github.com/lmagyar/homeassistant-addon-tailscale

**Reply by Ged-fi** [reply]
Any chance of a release reasonably soon? This change has been pending for quite a while.
